ClrMD.Extensions/ClrMDSession.cs

```diff
@@ -33,25 +33,19 @@ public class ClrMDSession : IDisposable
         public bool IsReferenceMappingCreated { get; private set; }
 
         internal ClrType ErrorType { get; private set; }
-
+        
         private ClrMDSession(DataTarget target, string dacFile)
         {
             ClrMDSession.Detach();
 
-            if (target.ClrVersions.Count == 0)
+            if (target.ClrVersions.Count() == 0)
                 throw new ArgumentException("DataTarget has no clr loaded.", nameof(target));
 
             Target = target;
             Runtime = dacFile == null ? target.ClrVersions[0].CreateRuntime() : target.ClrVersions[0].CreateRuntime(dacFile);
             Heap = Runtime.Heap;
 
-            //Temp hack until ErrorType is made public
-            var property = Heap.GetType().GetProperty("ErrorType", BindingFlags.Instance | BindingFlags.NonPublic);
-
-            if (property == null)
-                throw new InvalidOperationException("Unable to find 'ErrorType' property on ClrHeap.");
-
-            ErrorType = (ClrType)property.GetValue(Heap);
+            ErrorType = null; //I'm not sure if this is the right thing, but ErrorType doesn't exist in ClrMD 2.0
```

Comment:
This will probably break something, I think this hack was needed to match the `Type` of invalid objects.